static/css/restyle/restyle.less

```diff
@@ -111,11 +110,21 @@ hgroup h2.addon,
   line-height: 1.5;
 }
 
-masthead.submit-theme,
+#masthead.submit-theme,
 #collections-landing {
   margin-top: 20px;
 }
 
+#addon-description-header,
```

Comment:
ugh, this really is an id! :skull: Mind changing this, there's exactly two more occurences in:

* src/olympia/addons/templates/addons/impala/details.html
* site-static/js/impala/addon_details.js

imho. But again, baby dragons.